windows-msi/bookmarks/howto.url

```diff
@@ -0,0 +1,4 @@
+[DEFAULT]
+BASEURL=https://openvpn.net/howto.html
+[InternetShortcut]
+URL=https://openvpn.net/howto.html
```

Comment:
This may be a silly question, but why two sections with almost identical content?

Reply:
Ask Microsoft! :) This is how Internet Explorer authors the .url files. The .url file launches default web browser when opened by the shell. In other words: it's an URL version of a .lnk shortcuts.

Frankly, I didn't take time to test if the shell is happy with `URL` in `[InternetShortcut]` section alone. Wouldn't be surprised to see different behaviour on different Windows versions. So it seemed least collateral damage to leave both URL's in the file vs. testing all Windows versions and reverse engineering the file structure in the absence of official documentation.

Unfortunately, there is no official MSDN documentation on the .url files. The best description you can find online is [here](https://www.codeproject.com/articles/22267/internet-explorer-favorites-deconstructed). Reverse engineered by somebody.

Reply:
Ok. I don't mind, I was just wondering why :smile:.